src/Vester.sol

```diff
@@ -48,7 +49,7 @@ contract Vester is Initializable, IVester {
     event BeneficiaryChanged(address indexed oldBeneficiary, address indexed newBeneficiary);
     event VestingPositionCreated(uint256 indexed nonce, uint256 amount, uint256 vestingEnds);
     event Claimed(uint256 indexed nonce, uint256 amount);
-    event ClaimedAuraRewards(uint256 amount);
+    event ClaimedAuraRewards(uint256 amount, address indexed token);
```

Comment:
Why add a `token` arg here? The contract only supports claiming AURA. Similarly, the `Claimed` event does not include a token address.

Reply:
Done